Apps/FlightSoftware/Watchdog/src/watchdog.c

```diff
@@ -171,12 +171,12 @@ void watchdog_send_hercules_telem() {
         // [6,7] won't be able to read lander power in mission mode
         // [8] currently don't read battery temp in mission mode
         //telbuf[9] = (unit8_t)( sys_status (??) )
-        telbuf[10] = (uint8_t)(raw_battery_charge[0]);
-        telbuf[11] = (uint8_t)(raw_battery_charge[1]);
-        telbuf[12] = (uint8_t)(raw_battery_current[0]);
-        telbuf[13] = (uint8_t)(raw_battery_current[1]);
-        telbuf[14] = (uint8_t)(raw_battery_voltage[0]);
-        telbuf[15] = (uint8_t)(raw_battery_voltage[1]);
+        telbuf[10] = (uint8_t)(i2cReadings->raw_battery_charge[0]);
+        telbuf[11] = (uint8_t)(i2cReadings->raw_battery_charge[1]);
+        telbuf[12] = (uint8_t)(i2cReadings->raw_battery_current[0]);
+        telbuf[13] = (uint8_t)(i2cReadings->raw_battery_current[1]);
+        telbuf[14] = (uint8_t)(i2cReadings->raw_battery_voltage[0]);
+        telbuf[15] = (uint8_t)(i2cReadings->raw_battery_voltage[1]);
 //                telbuf[9] = (uint8_t)(adc_values[ADC_TEMP_IDX] >> 4);
```

Comment:
Nit: Remove dead code

Reply:
@ohnx should this be removed? Or will it be uncommented at some point

Reply:
@SchwartzCode merge in as in, I can deal with it later. I think what we will do is have a "debug" mode where we send down all this extra telem, and also a "in transit" mode where we have the reduced bandwidth and only send the essentials.